Documentation/Usage.md

```diff
@@ -146,81 +146,116 @@ following files from the libgit2 system-package are of interest:
     /usr/local/lib/libgit2.dylib      # .so on Linux
     /usr/local/include/git2.h
 
-Swift packages that provide module maps for system libraries are handled
-differently from regular Swift packages.
+**Note:** the system library may be located elsewhere on your system, such as
+`/usr/`, or `/opt/homebrew/` if you're using Homebrew on an Apple Silicon Mac.
+If you're not sure where `Clibgit` is in your system, you can run this command
+to look it up:
 
-Note that the system library may be located elsewhere on your system, such as
-`/usr/` rather than `/usr/local/`.
+    example$ pkg-config --cflags libgit2
+    -I/usr/local/libgit2/1.6.4/include
 
-Create a directory called `Clibgit` next to the `example` directory and
-initialize it as a package that builds a system module:
 
-    example$ cd ..
-    $ mkdir Clibgit
-    $ cd Clibgit
-    Clibgit$ swift package init --type empty
-
-This creates a `Package.swift` file in the directory.
-Edit `Package.swift` and add `pkgConfig` parameter:
+**First, let's define the `target` in the package description**:
 
 ```swift
-// swift-tools-version:5.3
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
-    name: "Clibgit",
-    pkgConfig: "libgit2"
+    name: "example",
+    targets: [
+        // systemLibrary is a special type of build target that wraps a system library
+        // in a target that other targets can require as their depencency.
+        .systemLibrary(
+            name: "Clibgit",
+            pkgConfig: "libgit2",
+            providers: [
+                .brew(["libgit2"]),
+                .apt(["libgit2-dev"])
```

Comment:
Perhaps add a `.nuget` entry as well?

Reply:
Hmmm. Should we suggest [`LibGit2Sharp`](https://www.nuget.org/packages/LibGit2Sharp)? Swift doesn't need the .Net wrapper to build it's package, but there doesn't seem to be another minimal nuget package for it.